chatbees/client_models/collection.py

```diff
@@ -1,12 +1,24 @@
 import os
-from typing import List, Dict, Tuple
+from typing import List, Dict, Tuple, Any, Union
 from urllib import request
 
 from pydantic import BaseModel
 
 from chatbees.client_models.chat import Chat
-from chatbees.client_models.doc import AnswerReference
+from chatbees.server_models.doc_api import (
+    CrawlStatus,
+    IngestionStatus,
+    AnswerReference,
+    SearchReference,
+)
 from chatbees.server_models.chat import ConfigureChatRequest, ChatAttributes
+from chatbees.server_models.ingestion_type import (
+    WebsiteSpec,
+    IngestionType,
+    NotionSpec,
```

Comment:
let's add Notion, GDrive later till the overwrite is supported. and Slack

Reply:
sg